RCTOpenTokPublisherView.m

```diff
@@ -55,8 +55,16 @@ - (void)mount {
  * inside the mounted view
  */
 - (void)startPublishing {
-    _publisher = [[OTPublisher alloc] initWithDelegate:self];
-
+    _publisher = [[OTPublisher alloc] initWithDelegate:self
+                                                  name: @"My Video"
+                                      cameraResolution: _cameraResolution
+                                       cameraFrameRate: (OTCameraCaptureResolution)_cameraFrameRate];
```

Comment:
ups!! this happens to me when things don't work!! I just modify things to see if something changes, it does not.. then I commit and push 😁